recipes/decord/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "decord" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/dmlc/{{ name }}.git
```

Comment:
Can we use an archive tag + checksum? The problem is that, on GH,  this is mutable and can cause problems later on. I understand that the submodules can be a challenge, but I think you can still check them out.

Reply:
Let me see if can I resolve it